### ISSUE TITLE
Training set of FGVC-Aircraft

### ISSUE THREAD

**Issue by ltong1130ztr** [PRIS-CV/Fine-Grained-or-Not#11]
> Split the dataset into train and test folder, the index of each class should follow the Birds.xls, Air.xls, and Cars.xls 

Hi, did you merge the training and validation set of FGVC-Aircraft into a new training set in your settings? Thank you!

**Comment by dongliangchang** [comment]
@RuoyiDu

**Comment by RuoyiDu** [comment]
> > Split the dataset into train and test folder, the index of each class should follow the Birds.xls, Air.xls, and Cars.xls
> 
> Hi, did you merge the training and validation set of FGVC-Aircraft into a new training set in your settings? Thank you!

Sorry for the late reply. Yes, we merged the train and val sets for model training, similar to many previous FGVC works (so, there are 6667 images for training and 3333 images for testing).
